static/js/components/tiles/chart_footer.tsx

```diff
@@ -81,6 +84,15 @@ export function ChartFooter(props: ChartFooterPropType): JSX.Element {
             Export
           </a>
         )}
+        {props.exploreMoreUrl && (
+          <a
+            href={props.exploreMoreUrl}
```

Comment:
Make sure the appropriate hostname `datacommons.org` / `autopush.datacommons.org` / `localhost` is explicitly added here (so this will work on other sites with web components)